python/GafferImageUI/ImageReaderUI.py

```diff
@@ -40,6 +40,14 @@
 import GafferUI
 import GafferImage
 
+def __colorSpacePresetNames( plug ) :
+
+	return IECore.StringVectorData( [ "Default" ] + GafferImage.OpenColorIOTransform.availableColorSpaces() )
+
+def __colorSpacePresetValues( plug ) :
+
+	return IECore.StringVectorData( [ "oiioDefault" ] + GafferImage.OpenColorIOTransform.availableColorSpaces() )
```

Comment:
This needs to be "" rather than "oiioDefault", so that it matches the actual default value of the plug, and the UI doesn't show the "Invalid" label.